### PR TITLE
Metal jaw

### PR DESCRIPTION
**Pull request by Chomski2**
## Changelog:
- Added a metal jaw for undead

## Developer changelog:
- Added this thing.
- Changed that thing.

## Tests:
- [ ] There are no new errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` 
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:

### CODE REVIEW

File: gfx/portraits/portrait_modifiers/05_headgear_situational.txt [hunk 1]
```diff
@@ -231,4 +231,45 @@
...
+		outfit_tags = { metal_jaw }
+		weight = {
+			base = 0
+			modifier = {
+				add = 5
+				has_trait = being_undead
+			}
+			modifier = {
+				factor = 0
+				has_trait = creature_high_elf
```
Comment (robmart) [inline gfx/portraits/portrait_modifiers/05_headgear_situational.txt]: trait check can be replaced by `culture ?= { has_cultural_pillar = heritage_highborne }`

File: gfx/portraits/portrait_modifiers/99_special.txt [hunk 2]
```diff
@@ -261,6 +261,25 @@ long_beard = {
 	}
 }
 
+no_jaw = {
```
Comment (robmart) [inline gfx/portraits/portrait_modifiers/99_special.txt:265]: Not marked with `#Warcraft`

Comment (robmart) [reply gfx/portraits/portrait_modifiers/99_special.txt:265]: Also no trigger on this weight, won't everyone be jaw-less?

File: gfx/portraits/portrait_modifiers/00_custom_special.txt [hunk 3]
```diff
@@ -141,4 +141,14 @@
 			has_no_portrait_trigger = no
 		}
 	}
+
+	add_accessory_modifiers = {
```
Comment (robmart) [inline gfx/portraits/portrait_modifiers/00_custom_special.txt:146]: Not marked with `#Warcraft`

**Review by robmart — CHANGES_REQUESTED** [review]

File: gfx/portraits/portrait_modifiers/99_special.txt (hunk 2 above)
Comment (Chomski2) [reply gfx/portraits/portrait_modifiers/99_special.txt:265]: This only applies if the accessory is marked with the "no_jaw" tag

Comment (robmart) [reply gfx/portraits/portrait_modifiers/99_special.txt:265]: Alright 👍

**Review by robmart — APPROVED** [review]
LGTM

**Pull request merged**